### PR TITLE
Add dbt utils package dependency to readme

### PR DESCRIPTION
**Pull request by sarah-tuva**
## Describe your changes

* Updated the README to include instructions for the dependent `dbt_utils` package.
* Fixed typo in the pull request template.

## How has this been tested?

Previewed markdown.

## Reviewer focus

Clarity and spelling in the README.

## Checklist before requesting a review

- [ ]  I have recorded a Loom performing a self-review of my code
- [ ]  My code follows style guidelines
- [ ]  I have commented my code as necessary
- [ ]  I have testing by code by running `dbt build` in the following warehouse(s):
    - [ ]  Snowflake
    - [ ]  Redshift
- [ ]  I have implemented generic dbt tests to validate primary keys/uniqueness in my model
- [ ]  I have updated dbt docs by running `dbt docs generate` and copying the appropriate files to the `docs/` path
- [x]  I have added at least one Github label to this PR

## For reviewers

[Best practices for reviewing pull requests.](https://www.notion.so/Pull-Request-Review-Best-Practices-b486354d76c04982889f2178fe777c1c)

### CODE REVIEW

**Comment by sarah-tuva** [comment]
Hey @tuvaforrest, this is the update I wanted to make for the CCLF connector. I'm making the same change in the SAF connector. Let me know if this overlaps with what you had planned and I can close this PR.

File: README.md [hunk 1]
```diff
@@ -17,8 +17,14 @@ For information on data models and to view the entire DAG check out our dbt [Doc
...
 ### Pre-requisites
-1. You have Medicare CCLF claims data loaded into a data warehouse
-2. You have [dbt](https://www.getdbt.com/) installed and configured (i.e. connected to your data warehouse)
+1. You have Medicare CCLF claims data loaded into a data warehouse.
+2. You have [dbt](https://www.getdbt.com/) installed and configured (i.e. connected to your data warehouse).
+3. This project is dependent on the package `dbt_utils`, include the following in your packages.yml:
+   ```
+   packages:
+     - package: dbt-labs/dbt_utils
+       version: [">=0.9.2"]
```
Comment (tuvaforrest) [inline README.md:26]: Do you think we should have this open ended version range?  1.0.0 was recently released, and I don't know if they affect us (haven't tested it) but I know there are some breaking changes in it.

In data profiling we have version: [">=0.9.2","<1.0.0"].  I think if we keep what's in the readme we should at least test on 1.0.0 first (maybe you've already done that @sarah-tuva ?)  If so maybe we change DP to be open ended like this one or at least bump up the major to version: [">=0.9.2","<2.0.0"], if not maybe we keep it as <1.0.0 for now?

**Review by tuvaforrest — COMMENTED** [review]
Left a comment, let me know your thoughts on the version range?

File: README.md (hunk 1 above)
Comment (sarah-tuva) [reply README.md]: @tuvaforrest I totally agree with keeping the range. I haven't done any testing with 1.0 yet. I just pushed an update to add the range.

**Review by tuvaforrest — APPROVED** [review]
Looks good.  I have some other changes coming up which will also touch the readme, will incorporate these changes.

**Comment by sarah-tuva** [comment]
> Looks good. I have some other changes coming up which will also touch the readme, will incorporate these changes.

@tuvaforrest Sounds good! Thanks for taking reviewing.

**Pull request merged**